src/components/comparePricing/comparePricing.js

```diff
@@ -0,0 +1,105 @@
+import React, { useState } from 'react'
+import stl from './comparePricing.module.scss'
+import PricingCard from './pricingCard'
+import {
+  Nigeria,
+  India,
+  Mexico,
+  CostaRica,
+  Colombia,
+  Dominican,
+} from 'assets/svg'
+
+const ComparePricing = () => {
+  const [data, setData] = useState('')
```

Comment:
Here instead of blank data we can add the initial data state to be the first country of the object so that default values are selected. and then update it onClick().